de.metas.acct.base/src/main/sql/postgresql/ddl/functions/SummaryAndBalanceListReport.sql

```diff
@@ -0,0 +1,66 @@
+--- backup for erpit
+CREATE INDEX fas_dateacct
+    ON fact_acct_summary (dateacct);
+DROP INDEX fas_dateacct;
+-- todo @ teo observation: it seems the query runs faster without the index fas_dateacct. (27 sec avg -> 22 sec avg => >10% perf change)
+-- todo extract the index drop to own migration script
+--------------------------------------
+
+
+CREATE OR REPLACE FUNCTION SummaryAndBalanceListReport(p_dateFrom date,
+                                                       p_dateTo date,
+                                                       p_c_acctschema_id NUMERIC,
+                                                       p_ad_org_id numeric,
+                                                       p_account_id NUMERIC=NULL,
+                                                       p_c_activity_id numeric=NULL)
+    RETURNS table
+            (
+                AccountValue     text,
+                AccountName      text,
+                beginningBalance numeric,
+                debit            numeric,
+                credit           numeric,
+                endingBalancw    numeric
+            )
+AS
+$$
+WITH filteredElementValues AS
+         (
+             SELECT ev.c_elementvalue_id,
+                    ev.name  AccountName,
+                    ev.value AccountValue
+             FROM c_elementvalue ev
+             WHERE TRUE
+               AND (p_account_id IS NULL OR ev.c_elementvalue_id = p_account_id)
+               AND (p_c_activity_id IS NULL OR ev.c_activity_id = p_c_activity_id)
```

Comment:
activity filtering shall be applied on Fact_Acct, Fact_Acct_Summary.
If we don't have the Fact_Acct_Summary.C_Activity_ID then pls:
* skip it
* inform Mark 
* create a follow-up to introduce it

Reply:
> create a follow-up to introduce it

discussed with @teosarca - there's no need to introduce it


> skip it

Will do

Reply:
no longer filtering by activity_id.